projects/crc-timing/visualizations/crc-timing/plotting/init.py

```
@@ -0,0 +1,314 @@
+        "cloud-init.service",
+        "kubelet.service",
+    ]
+    for entry in entries:
+        try:
+            vm_start_time = entry.results.systemd_units["system.slice"]['Active Enter Timestamp']
+        except KeyError:
+            logging.error(f"'system.slide' not found, cannot generate the init services data for entry {entry.settings}")
+            continue
+
+        for name, unit in entry.results.systemd_units.items():
+            if "target" in name: continue
```

Ruff (0.13.1)

Multiple statements on one line (colon)

(E701)